machineLearning/sentimentAnalysis/twitterSentimentAnalysis/databaseUpdate/README.md

```diff
@@ -0,0 +1,13 @@
+# Database Update
+
+Similar to the news sentiment database update section
+The script `main.py` is used to update the tweets already in the database
+
+## Choosing model
+
+The script expects a `.env` file within this directory.
+In the `.env` file are 3 parameters.
+
+1. MODEL_DIR - The base directory where the model is trained
+2. MODEL_NAME - The name of the model e.g. NaiveBayesClassifier
+3. MONGO_URI - The Mongo uri for the database to update.
```

Comment:
All good, if you want to add just a section under this what model is being used now might be good idea actually just in case anyone runs it and overwrites it, might do same for my readme.